### PR TITLE
Add alias

### PR DESCRIPTION
**Pull request by ChrysanthemumT**
no delete functionality yet
just most basic aliasing

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddAliasCommand.java
Comment (cyqjoseph) [inline src/main/java/seedu/address/logic/commands/AddAliasCommand.java:1]: Great addition, however can you add some test cases for code coverage!

**Review by cyqjoseph — COMMENTED** [review]
Overall great job, can just include some test coverages and we should be good to go!

File: src/main/java/seedu/address/logic/commands/AddAliasCommand.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    public static final String MESSAGE_SUCCESS = "New Alias added: %1$s";
+
+    private final String alias;
+    private final String toReplace;
+
+    /**
+     * Creates an AddCommand to add the specified {@code Person}
+     */
+    public AddAliasCommand(String alias, String toReplace) {
```
Comment (gauravuj) [inline src/main/java/seedu/address/logic/commands/AddAliasCommand.java:25]: This is a very much needed addition - it significantly enhances the functionality for our target users, especially masters of CLI

**Review by gauravuj — COMMENTED** [review]
Overall, a great improvement - just need to add some test cases which can be a little tedious. Great work!

**Review by jerremyng — APPROVED** [review]
LGTM

**Pull request merged**